bors.toml

```
@@ -1,6 +1,5 @@
 status = [
-  "buildkite/cardano-wallet-post-commit", 
-  "buildkite/cardano-wallet-pre-merge"
+  "buildkite/cardano-wallet-pipeline", 
```

👍 I managed to unarchive the old pipeline under just the name [cardano-wallet](https://buildkite.com/input-output-hk/cardano-wallet) though:

```suggestion
  "buildkite/cardano-wallet", 
```